### PR TITLE
(DOCSP-27327) Update user metadata SDK links

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

This PR adds several SDK links  to the User Metadata page. It should be merged _after_ the new Node.js SDK page for [user meta data PR](https://github.com/mongodb/docs-realm/pull/2503) is merged.

### Jira

- https://jira.mongodb.org/browse/DOCSP-27327

### Staged Changes

- [Enable User Metadata](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-27372/users/enable-user-metadata/#access-user-metadata-from-a-client-application)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/users/enable-user-metadata.txt [hunk 1]
```diff
@@ -226,13 +226,10 @@ Procedure
 Access User Metadata from a Client Application
```
Comment (cbullinger) [inline source/users/enable-user-metadata.txt:226]: A couple blank lines above this heading  - are they there on purpose?

File: source/users/enable-user-metadata.txt [hunk 2]
```diff
@@ -226,13 +226,10 @@ Procedure
...
-         
-         Learn how to :ref:`read a user's metadata from an iOS
-         application <ios-read-user-metadata>`.
+Realm SDKs have different implementations for accessing user metadata in a client
+application. Check out the details in the SDK documentation:
 
+- :ref:`Realm Swift SDK <ios-read-user-metadata>`
+- :ref:`Realm Flutter SDK <flutter-read-user-metadata>`
+- :ref:`Realm .NET SDK <dotnet-read-user-metadata>`
+- :ref:`Realm Node.js SDK <ios-read-user-metadata>`
```
Comment (cbullinger) [inline source/users/enable-user-metadata.txt:235]: is this Realm Node.js SDK link purposefully pointing to a duplicate `<ios-read-user-metadata>`?

**Review by cbullinger — APPROVED** [review]

File: source/users/enable-user-metadata.txt (hunk 1 above)
Comment (krollins-mdb) [reply source/users/enable-user-metadata.txt:226]: Not that I know of! I'll clean that up.

File: source/users/enable-user-metadata.txt (hunk 2 above)
Comment (krollins-mdb) [reply source/users/enable-user-metadata.txt:235]: lol. No. I thought I fixed that, but clearly didn't. Thanks for pointing it out!

File: source/users/enable-user-metadata.txt (hunk 1 above)
Comment (krollins-mdb) [reply source/users/enable-user-metadata.txt:226]: Now that I'm looking, there are extra lines all over in this file...

**Pull request merged**